packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -185,7 +198,9 @@ export default abstract class SqlIntegration
           )}
           GROUP BY
             experiment_id,
+            ${groupByExperimentNameCol},
```

Comment:
Don't modify the `GROUP BY` clause at all.  Since you are doing an aggregate above (`MIN`), you don't need to also add the column to the group by.

Reply:
Resolved by new changes